docs/source/commands/peerlifecycle.md

```diff
@@ -595,7 +595,7 @@ also outputs which organizations have approved the chaincode definition. If an
 organization has approved the chaincode definition specified in the command, the
 command will return a value of true. You can use this command to learn whether enough
 channel members have approved a chaincode definition to meet the
-`/Channel/Application/Endorsement` policy (a majority by default) before the
```

Comment:
Did you remove this with a batch search-replace?

Reply:
It's related because it's the output of the automated doc generation for command help. It's clear that commands have been getting updated without regenerating the docs. The regen of the docs was explicitly called out in the commit message.

In this case, commit 49ed882a28 changed the postscript without updating docs.